lms/templates/components/card/card.underscore

```diff
@@ -2,40 +2,40 @@
 <div class="wrapper-card-core">
     <div class="card-core">
         <% if (pennant) { %>
-            <small class="card-type"><%- pennant %></small>
+            <small class="card-type"><%= HtmlUtils.escape(pennant) %></small>
         <% } %>
         <h3 class="card-title"
             <% if (!_.isUndefined(srInfo)) { %>
                 aria-describedby="<%= srInfo.id %>"
             <% } %>
-            ><%- title %>
+            ><%= HtmlUtils.escape(title) %>
         </h3>
-        <p class="card-description"><%- description %></p>
+        <p class="card-description"><%= HtmlUtils.escape(description) %></p>
```

Comment:
Can you confirm that all of these (see below) are wrapped this way because the code is using the variable (in this case description) as either HTML or plain text?  
If it is always HTML, leave as is but rename to descriptionHtml.
If it could be either, leave as is.
If it is always plain text, revert to `<%-`.